Example/StripeConnectExample/StripeConnectExample.xctestplan

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
I added this because we might want tests in the future (unsure) but it has the added benefit of allowing us to add it as a test run in Bit rise to ensure the app builds.

Reply:
Good call – I think it is important to at least ensure the app builds in Bitrise